roles/create-ai-environment/tasks/main.yaml

```diff
@@ -0,0 +1,278 @@
+      fi
+
+      if [ $runtime_name == 'go' ];then
+        apt install -y golang-$runtime_version-go
+      fi
+
+    executable: /bin/bash
+
+- name: Prepare GPU env (GPU driver and CUDA)
+  when: is_gpu_enabled
+  shell:
+    cmd: |
+      set -xe
+      apt-key adv --fetch-keys http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/7fa2af80.pub
+      wget http://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/cuda-repo-ubuntu1604_9.1.85-1_amd64.deb
+      cd /var/
+      apt purge -y cuda*
+      apt-get remove -y --purge nvidia-418 nvidia-modprobe nvidia-settings
+      cd -
+      apt install -y ./cuda-repo-ubuntu1604_9.1.85-1_amd64.deb
+      wget http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64/nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb
+      apt install -y ./nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb
+      apt update
+      apt install -y cuda9.0 cuda-cublas-9-0 cuda-cufft-9-0 cuda-curand-9-0 cuda-cusolver-9-0 cuda-cusparse-9-0 libcudnn7=7.2.1.38-1+cuda9.0 libnccl2=2.2.13-1+cuda9.0 cuda-command-line-tools-9-0
+      apt install -y linux-headers-4.4.0-142-generic
+      wget http://us.download.nvidia.com/tesla/384.81/NVIDIA-Linux-x86_64-384.81.run
+      chmod +x ./NVIDIA-Linux-x86_64-384.81.run
+      cat > 'input' <<EOF
+      yes
+      yes
+      EOF
+      ./NVIDIA-Linux-x86_64-384.81.run -a --ui=none < ./input
+    executable: /bin/bash
+
+- name: Install {{ user_meta.framework.name }}-{{ user_meta.framework.version }}
+  when:
+    ( user_meta.framework.name == "tensorflow" and runtime_name.stdout in ["python", "c", "java", "go"] ) or
+    ( user_meta.framework.name == "pytorch" and runtime_name.stdout in ["python", "c"] )
+  shell:
+    cmd: |
+      set -x
+      set -e
+      framework={{ user_meta.framework.name }}
+      framework_version={{ user_meta.framework.version }}
+      runtime_name={{ runtime_name.stdout }}
+
+      # TODO(wxy): And and test all the runtime. Now only tensorflow is tested.
+
+      if [ $framework == "tensorflow" ];then
+        if [ $runtime_name == "python" ];then
+          if {{ is_gpu_enabled | lower }} ;then
+            pip install tensorflow-gpu==$framework_version
+          else
+            pip install tensorflow==$framework_version
+          fi
+        fi
+        if [ $runtime_name == "java" ];then
+          wget https://storage.googleapis.com/tensorflow/libtensorflow_jni-cpu-linux-x86_64-$framework_version.tar.gz
+          tar -xvf libtensorflow_jni-cpu-linux-x86_64-$framework_version.tar.gz
+        fi
+        if [ $runtime_name == "go" ];then
+          mkdir ./download
+          export GOPATH=`pwd`/download
+          apt install -y git-core
+          go get github.com/tensorflow/tensorflow/tensorflow/go
+          go test github.com/tensorflow/tensorflow/tensorflow/go
+          git checkout tags/$framework_version
```

Comment:
I think you should change directory to tensorflow/go, then checkout specific version, then go test